code/__DEFINES/rust_g.dm

```diff
@@ -6,6 +6,7 @@
 #define RUSTG_JOB_ERROR "JOB PANICKED"
 
 #define rustg_dmi_strip_metadata(fname) call(RUST_G, "dmi_strip_metadata")(fname)
+#define rustg_dmi_create_png(fname,width,height,data) call(RUST_G, "dmi_create_png")(fname,width,height,data)
```

Comment:
bit of a nitpick but can you link to the rustg PR in PRs that update rustg so I know what to look at?

Reply:
https://github.com/tgstation/rust-g/pull/21
https://github.com/tgstation/rust-g/pull/22